ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,430 @@
+---
+eip: to-be-assign
```

Comment:
```suggestion
eip: 7818
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.